src/brsTypes/components/RoSGNode.ts

```diff
@@ -1323,7 +1323,7 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
                 let childNodesElements = child_nodes.getElements();
                 if (childNodesElements.length !== 0) {
                     childNodesElements.forEach((childNode) => {
-                        if (childNode instanceof RoSGNode) {
+                        {
```

Comment:
Can we remove these brackets?

Reply:
`if (childNode instanceof RoSGNode)` was deleted accidentally, I didn't intend to change the `append children` method 🤭 I am putting the entire line back in.